### ISSUE TITLE
Adding the White Box component to an entity for the first time crashes the Editor

### ISSUE THREAD

**Issue by LB-TomaszSerwanski** [o3de/o3de#12221]
**Describe the bug**
The Editor crashes after the White Box component is added to an entity for the first time on a project. 
Please note that the crash does not occur after launching the Editor again on the same project.

Please refer to the attached video and log files for more details.

**Steps to reproduce**
1. Create and build a new project.
2. Launch the Editor and create a new level.
3. Create a new entity and add a White Box component to it.

**Expected behavior**
White Box component is added to an entity. The Editor remains stable.

**Actual behavior**
The Editor crashes.

**Screenshots/Video**

https://user-images.githubusercontent.com/86953108/192527167-ac55d006-2dba-401a-a69a-d8f49f76c1ac.mp4

**Found in Branch**
Development (134fe32)
Stabilization/2210 (b83f535)

**Desktop/Device:**
 - Device: PC
 - OS: Windows
 - Version 10
 - CPU AMD Ryzen 5 3600
 - GPU Nvidia RTX 2060 SUPER
 - Memory 16GB

**Attachments**
[Editor.log](https://github.com/o3de/o3de/files/9655799/Editor.log)
[error.log](https://github.com/o3de/o3de/files/9655800/error.log)
[error.zip](https://github.com/o3de/o3de/files/9655801/error.zip)

**Comment by hultonha** [comment]
@LB-TomaszSerwanski would it please be possible to quickly test this with EditorModeVisualFeedback disabled (this can be done in the settings registry). Let me know if you need instructions on how to do this.

@jonawals will be taking a look shortly

**Comment by jonawals** [comment]
Fixed here: https://github.com/o3de/o3de/pull/12226

**Comment by LB-TomaszSerwanski** [comment]
@hultonha yes, disabling the EditorModeFeedback in setreg file fixes the issue. Will verify again with the default settings once the above PR has been merged into the branch.

**Comment by hultonha** [comment]
Awesome thank you @LB-TomaszSerwanski, I've just merged the branch (https://github.com/o3de/o3de/pull/12226) so hopefully you should be able to verify shortly. Thanks!

**Comment by LB-TomaszSerwanski** [comment]
@hultonha, unfortunately, the crash still occurs on the latest Stabilization/2210 (943ea1f) with exactly the same logs (attaching them again just in case). I double checked that the changes from the #12226 are in fact present in the build. 

[Editor.log](https://github.com/o3de/o3de/files/9665199/Editor.log)
[error.log](https://github.com/o3de/o3de/files/9665200/error.log)
[error.zip](https://github.com/o3de/o3de/files/9665201/error.zip)

**Comment by hultonha** [comment]
Thanks for the update @LB-TomaszSerwanski, can you please look into this again @jonawals, it seems like this is still causing a problem. Thanks!

**Comment by hultonha** [comment]
@LB-TomaszSerwanski I just caught up with @jonawals and as the stabilization branch is red it's possible you don't have the latest build with this change (unless you built from source?) Can you please confirm? Thanks!

**Comment by hultonha** [comment]
@LB-TomaszSerwanski have you had a chance to try this again sorry with the latest green development build? Is it still happening? Thanks!

**Comment by LB-TomaszSerwanski** [comment]
@hultonha sorry for delay, but I wanted to verify on both source engine as well as on the pre-built SDK. The crash occurs on both setups on the latest stabilization/2210 (67014ed). We have used the steps from the [official guide](https://www.o3de.org/docs/welcome-guide/setup/setup-from-github/building-windows/) after wiping the following folders out of the machine:
- C:/git/o3de/build
- C:/o3de-install
- C:/users/<username>/.o3de
- C:/users/<username>/O3DE

There is also one machine on which we have deleted all the files related to o3de and performed the fresh O3DE repository clone and source engine build, however the project hasn't yet finished building there. Will update further once that is done.

**Comment by hultonha** [comment]
Thanks for the update @LB-TomaszSerwanski. This is really odd as with the latest change @jonawals has not been able to reproduce this. It would be great to triple check you definitely have the fix in https://github.com/o3de/o3de/pull/12226. Do please reach out to me and/or @jonawals for more info tomorrow. Thanks!

**Comment by santorac** [comment]
This issue might be related to https://github.com/o3de/o3de/issues/12253. I noted similar repro steps here https://github.com/o3de/o3de/issues/12253#issuecomment-1263172647. However, I'm not sure whether my repro steps deviate from the original bug being reported in 12253. I will continue to look into this.

**Comment by hultonha** [comment]
We have discovered this is most likely to do with a shader/material asset not being ready when the editor first launches with a new project. We've been working with various teams to try and find the best way to handle this (ideally we'd like to make the shader/material a critical asset but apparently this isn't possible). I am aware both @amzn-mike and @jonawals are discussing it and hopefully we should have a fix first thing next week. We were unfortunately not able to resolve it today. It can be worked around in the short-term by disabling Editor Mode Visual Feedback in the settings registry (if you think this is causing an issue in Automated Testing then we could set that for that project perhaps?)

**Comment by santorac** [comment]
Okay, it's probably not the same thing then.

**Comment by tonybalandiuk** [comment]
** Exception Tracking**

* [ ] 1. Owning SIG requests the exception, by a Sig Representative (Chair/co-chair) adding the PR to the exception requests queue board.
* [ ] 2. Owning SIG comments on the PR explaining impact if this code does not go into stabilization along with the blast radius of this code.
* [ ] 3. Unrelated SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
* [ ] 4. UX SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
* [ ] 5. Testing SIG representative (chair/cochair) approves the exception, adding a comment indicating approval and the SIG they represent.
* [ ] (Must occur at some point before SIG-Release approval) Code has been approved for merging by maintainers
* [ ] 6. SIG-Release Release Manager or co-Release Manager approves the exception, adding a comment indicating approval and the SIG they represent.
* [ ] 7. If all approvals are accepted, then the exception PR may be merged
* [ ] 8. Merged PR is verified successful via testing. If Merged PR verification testing failed, consult reps from step 3-5 and discussion occurs on fixing the failure vs backing out change
* [ ] 9. Exception is complete

**Comment by tonybalandiuk** [comment]
The PR for this will need an exception @hultonha  . Can you see that it gets prepared with the exception checklist? Thanks.

**Comment by hultonha** [comment]
@tonybalandiuk see https://github.com/o3de/o3de/pull/12364 for PR and exception checklist

**Comment by hultonha** [comment]
Fixed by https://github.com/o3de/o3de/pull/12364

**Comment by LB-TomaszSerwanski** [comment]
The issue verified as Fixed on the latest Stabilization/2210 (0770a89) branch.
